docs/format_description.rst

```diff
@@ -4,7 +4,11 @@
 The Notebook file format
 ========================
 
-Some general points about the notebook format:
+The official Jupyter Notebook format is defined with
+`this JSON schema <https://github.com/jupyter/nbformat/blob/master/nbformat/v4/nbformat.v4.schema.json>`_,
+which is used by Jupyter tools to validate notebooks.
+
+This page contains some general points about the notebook format.
```

Comment:
```suggestion
This page contains a human-readable description of the notebook format.
```

I think we can be a bit more optimistic now than 'some general points' :slightly_smiling_face:

Reply:
good point :-)